src/status_im/transport/message/transit.cljs

```diff
@@ -92,6 +92,8 @@
                               "c3" (fn [[name profile-image address fcm-token]]
                                      (v1.contact/ContactRequestConfirmed. name profile-image address fcm-token))
                               "c4" (fn [[content content-type message-type clock-value timestamp]]
+                                     (v1.protocol/Message. (if (string? content) {:text content} content) content-type message-type clock-value timestamp))
```

Comment:
To be honest, it's not so easy, because for example we already have messages (command content type) where content is a map (and doesn't contain text).